### DISCUSSION TITLE
fieldparams - more info

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16564] [How to]
Looking for guidance on how to control single fields with fieldparams entries

Appreciate help

**Comment by mariuszkrzaczkowski** [comment]
there is no description / documentation, you have to look at the code, each field may have different configuration parameters

**Reply by mariuszkrzaczkowski** [reply]
https://github.com/YetiForceCompany/YetiForceCRM/search?q=getFieldParams
